pkg/apis/sentry/v1alpha1/zz_generated.openapi.go

```diff
@@ -11,13 +11,13 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"github.com/markturansky/sentry-operator/pkg/apis/sentry/v1.Sentry":       schema_pkg_apis_sentry_v1_Sentry(ref),
-		"github.com/markturansky/sentry-operator/pkg/apis/sentry/v1.SentrySpec":   schema_pkg_apis_sentry_v1_SentrySpec(ref),
-		"github.com/markturansky/sentry-operator/pkg/apis/sentry/v1.SentryStatus": schema_pkg_apis_sentry_v1_SentryStatus(ref),
+		"github.com/sd-hackday-sentry/sentry-operator/pkg/apis/sentry/v1alpha1.Sentry":       schema_pkg_apis_sentry_v1alpha1_Sentry(ref),
+		"github.com/sd-hackday-sentry/sentry-operator/pkg/apis/sentry/v1alpha1.SentrySpec":   schema_pkg_apis_sentry_v1alpha1_SentrySpec(ref),
+		"github.com/sd-hackday-sentry/sentry-operator/pkg/apis/sentry/v1alpha1.SentryStatus": schema_pkg_apis_sentry_v1alpha1_SentryStatus(ref),
```

Comment:
@markturansky you may be interested. The script generated code with your repo. I guess you ran it under your repo or something else happened.